lib/inspec/objects/test.rb

```diff
@@ -67,7 +67,7 @@ def rb_describe
       itsy = xtra.nil? ? 'it' : 'its(' + xtra.to_s.inspect + ')'
       naughty = @negated ? '_not' : ''
       xpect = defined?(@expectation) ? expectation.inspect : ''
-      if matcher == 'match'
+      if @expectation.class == Regexp
```

Comment:
Great improvement, to cover regular expression handling for all kind of matchers